docs/blog/articles/2024-05-15-Kserve-0.13-release.md

```diff
@@ -0,0 +1,75 @@
+# From Serverless Predictive Inference to Generative Inference: Introducing KServe v0.13
+
+We are excited to unveil KServe v0.13, marking a significant leap forward in providing features for the evolving demands of model serving. This release is highlighted by three pivotal updates: enhanced Huggingface runtime, robust vLLM support, and the integration of OpenAI protocol standards.
+
+Here is a summary of the key changes:
+
+## Enhanced HuggingFace Runtime Support
+
+KServe v0.13 enriches its Huggingface runtime and now supports running HuggingFace models out-of-the-box. KServe v0.13 implements a [KServe HuggingFace Serving Runtime](https://github.com/kserve/kserve/tree/master/python/huggingfaceserver), `kserve-huggingfaceserver`. With this implementation, KServe can now automatically infer a [task](https://huggingface.co/tasks) from model architecture. Currently supported tasks include sequence classification, token classification, fill mask, text generation, and text to text generation. 
+
+Here is an example Inference Service with HuggingFace runtime. See more details in our updated docs to [Deploy the Llama2 model with Hugging Face LLM Serving Runtime](https://kserve.github.io/website/master/modelserving/v1beta1/llm/huggingface/) and our python SDK [HuggingFace Server Runtime docs](https://github.com/kserve/kserve/tree/master/python/huggingfaceserver).
+
+```yaml
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: huggingface-llama2
+spec:
+  predictor:
+    model:
+      modelFormat:
+        name: huggingface
+      args:
+      - --model_name=llama2
+      - --model_id=meta-llama/Llama-2-7b-chat-hf
+      resources:
+        limits:
+          cpu: "6"
+          memory: 24Gi
+          nvidia.com/gpu: "1"
+        requests:
+          cpu: "6"
+          memory: 24Gi
+          nvidia.com/gpu: "1"
+```
+
+### vLLM support 
+
+Version 0.13 introduces dedicated runtime support for [vLLM](https://docs.vllm.ai/en/latest/), for enhanced transformer model serving. This support now includes auto-mapping vLLMs as the backend for supported tasks, streamlining the deployment process and optimizing performance.
```

Comment:
Also talk about the case off fallbacking to HF if vllm is not available or doesn't support the model architecture

Reply:
added